neoexchange/core/views.py

```diff
@@ -3743,37 +3745,41 @@ def plot_all_spec(source):
     return script, div, p_spec
 
 
-def plot_floyds_spec(block, obs_num=1):
+def plot_floyds_spec(block):
     """Get plots for requested blocks of FLOYDs data and subtract nearest solar analog."""
 
     date_obs, obj, req, path, prop = find_spec(block.id)
     filenames = search(path, matchpattern='.*_2df_ex.fits', latest=False)
     if filenames is False:
-        return '', {"raw_spec": ''}
+        return None, None
     filenames = [os.path.join(path, f) for f in filenames]
 
     analogs = find_analog(block.when_observed, block.site)
 
     try:
-        raw_label, raw_spec, ast_wav = spectrum_plot(filenames[obs_num-1])
-        data_spec = {'label': raw_label,
-                     'spec': raw_spec,
-                     'wav': ast_wav,
-                     'filename': filenames[obs_num-1]}
+        data_spec = []
+        for filename in filenames:
+            raw_label, raw_spec, ast_wav = spectrum_plot(filename)
+            data_spec.append({'label': raw_label,
+                              'spec': raw_spec,
+                              'wav': ast_wav,
+                              'filename': filename})
     except IndexError:
         data_spec = None
 
     analog_data = []
-    offset = 0
+    offset = 2
```

Comment:
Add a comment documenting what this `offset` magic number is ? Add `offset` explanation into doc string for `spectrum_plot` ?